src/lib/tasks/chunk_migration_task.cpp

```diff
@@ -0,0 +1,66 @@
+#include "chunk_migration_task.hpp"
+
+#if OPOSSUM_NUMA_SUPPORT
+
+#include <iostream>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "storage/numa_placement_manager.hpp"
+#include "storage/storage_manager.hpp"
+#include "storage/table.hpp"
+
+namespace opossum {
+
+ChunkMigrationTask::ChunkMigrationTask(const std::string& table_name, const ChunkID chunk_id, int node_id)
```

Comment:
I think I would only provide one interface. If the caller has only one ChunkID, it should use {}.